.templates/hassio/service.yml

```diff
@@ -0,0 +1,15 @@
+  hassio:
+    image: "homeassistant/home-assistant:latest"
+    container_name: "hassio"
+    restart: unless-stopped
+    network_mode: "host"
+    depends_on:
+      - influxdb
+      - mosquitto
+    ports:
```

Comment:
Port is technically not required since `network_mode` is set to `"host"`, but I left it in just in case people want to disable host networking mode.